common/views/components/InputTypeNumber/InputTypeNumber.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import { forwardRef, type ComponentType } from 'react';
+import styled from 'styled-components';
+import Space, { type SpaceComponentProps } from '../styled/Space';
+import { classNames } from '../../../utils/classnames';
+
+const StyledInput: ComponentType<SpaceComponentProps> = styled(Space).attrs({
```

Comment:
Why not just change the type here instead of [here](https://github.com/wellcometrust/wellcomecollection.org/pull/4770/files#diff-4b6fcbe430c6c9136f18d988f992741eR41)?